terraform/service_rotation.tf

```diff
@@ -223,3 +223,30 @@ resource "google_cloud_scheduler_job" "rotation-worker" {
     google_project_service.services["cloudscheduler.googleapis.com"],
   ]
 }
+
+resource "google_cloud_scheduler_job" "realm-key-rotation-worker" {
+  name             = "realm-key-rotation-worker"
+  region           = var.cloudscheduler_location
+  schedule         = "2,32 * * * *"
```

Comment:
2,32... those are oddly specific minutes

Reply:
offsetting it from the token rotation (which is every 5 minutes)